clusterloader2/testing/request-benchmark/modules/measurements.yaml

```diff
@@ -0,0 +1,25 @@
+## Measurement module defines test scoped measurement.
+
+## Input params
+# Valid actions: "start", "gather"
+{{$action := .action}}
+
+steps:
+- name: "{{$action}}ing measurements"
+  measurements:
+  - Identifier: ContainerCPU
+    Method: GenericPrometheusQuery
+    Params:
+      action: {{$action}}
+      metricName: Container CPU
+      metricVersion: v1
+      unit: cores
+      dimensions:
+      - container
+      queries:
+      - name: Perc99
+        query: quantile_over_time(0.99, sum by (container) (rate(container_cpu_usage_seconds_total[1m]))[%v:])
```

Comment:
The open question (we can address it here or in the next iteration [e.g. when the test is already running]):

- What should be a desired unit of result? Now you return CPU/s, Personally I would vote for "CPU/s per inflight" as this is the number we are looking for.